### PR TITLE
Add aerospike 3.8.2.2 build

### PR DESCRIPTION
**Pull request by roa**

### CODE REVIEW

**Comment by ferki** [comment]
Please mark the ebuild as beta by appending `_beta` to the package name (ref: [Ebuild File Format](https://devmanual.gentoo.org/ebuild-writing/file-format/index.html)), otherwise LGTM :+1:

File: dev-db/aerospike-server-enterprise/aerospike-server-enterprise-3.7.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+RDEPEND="app-crypt/gcr"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${P}-debian7"
+
+RESTRICT="fetch"
+
+# change me at every version bump
+TOOLS_PV="3.7.2"
```
Comment (ferki) [inline dev-db/aerospike-server-enterprise/aerospike-server-enterprise-3.7.4.ebuild]: Looks like it was forgotten to be bumped for 3.7.4.

**Comment by ferki** [comment]
I fixed merge conflicts, tools version and ebuild name.

I'm not sure if this is still in beta, or the ebuild is still needed, but at least it should be here to grab later.

**Comment by roa** [comment]
i changed the pr title. it is now an update to 3.8.2.2
but it is still based on the ebuild we wanted to use for beta

**Comment by ferki** [comment]
It's still not marked as `_beta` in the ebuild name (I think you can just simply rename it, and add the bits from my beta version earlier).

**Comment by roa** [comment]
3.8.2.2 is stable

**Comment by ferki** [comment]
In that case, LGTM :+1: (the issue title confused me :smile: ) I can prepare the beta next time, just ping me.

**Comment by roa** [comment]
fixed the title

**Comment by ferki** [comment]
Oh, you can ignore the failed test, as I merged a passing version before rebasing the branch and pushing it again, but since I was too fast to merge, travis couldn't find this branch anymore to test.

**Pull request merged**